### PR TITLE
List view activity

### PR DESCRIPTION
**Pull request by FoulgerDevin**
Implements the list view so that the search results are displayed properly.

### CODE REVIEW

File: UniversityGear/.idea/inspectionProfiles/Project_Default.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<component name="InspectionProjectProfileManager">
```
Comment (rcgonzalezf) [inline UniversityGear/.idea/inspectionProfiles/Project_Default.xml:1]: I won't recommend tracking this file with Git, try to add a gitignore and remove this metadata IDE files from the repo.

File: UniversityGear/.idea/misc.xml [hunk 2]
```diff
@@ -43,4 +43,20 @@
   <component name="ProjectType">
     <option name="id" value="Android" />
   </component>
+  <component name="masterDetails">
```
Comment (rcgonzalezf) [inline UniversityGear/.idea/misc.xml:46]: ditto.

File: UniversityGear/.idea/inspectionProfiles/profiles_settings.xml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<component name="InspectionProjectProfileManager">
```
Comment (rcgonzalezf) [inline UniversityGear/.idea/inspectionProfiles/profiles_settings.xml:1]: ditto

File: UniversityGear/.idea/inspectionProfiles/Project_Default.xml (hunk 1 above)
Comment (FoulgerDevin) [reply UniversityGear/.idea/inspectionProfiles/Project_Default.xml:1]: Got it, thanks!

**Pull request merged**